neurom/features/tests/test_neuron_features.py

```diff
@@ -2,19 +2,78 @@
 import math
 import numpy as np
 from nose import tools as nt
+from neurom.core.tree import Tree
+from neurom.core.neuron import make_soma
 from neurom.features import neuron_features as nf
+from neurom.core.types import TreeType
 from neurom.ezy import load_neuron
 
 _path = os.path.dirname(os.path.abspath(__file__))
 DATA_PATH = os.path.join(_path, '../../../test_data')
 SWC_PATH = os.path.join(DATA_PATH, 'swc')
 
+class MockNeuron: pass
+
 NEURON_PATH = os.path.join(SWC_PATH, 'Neuron.swc')
 NEURON = load_neuron(NEURON_PATH)
+NEURONS = [NEURON, NEURON]
+
+def test_soma_radii():
+    nt.eq_(list(nf.soma_radii(NEURONS)), [0.17071067811865476, 0.17071067811865476])
+
+def test_soma_surface_areas():
+    area = 4. * math.pi * list(nf.soma_radii(NEURON))[0] ** 2
+    nt.eq_(list(nf.soma_surface_areas(NEURONS)), [area, area])
+
+def test_trunk_origin_elevations():
```

Comment:
Could you test empty sets too? i.e. use a tree type that isn't present in the neuron.